website/docs/r/storage_bucket.html.markdown

```diff
@@ -146,4 +146,4 @@ Storage buckets can be imported using the `name`, e.g.
 $ terraform import google_storage_bucket.image-store image-store-bucket
 ```
 
-Note that when importing a bucket (and only when importing), the Compute API needs to be enabled - you'll see an error with a link to the enablement page if it is not.
+**Note**: When importing a bucket (and only when importing), the Compute API needs to be enabled - you'll see an error with a link to the enablement page if it is not. You may also need to enable the Compute API if you are using the default provider project and not setting `project` on the bucket resource.
```

Comment:
It's "will", not "may" - I'd rephrase to "If you would prefer not to enable the Compute API, ensure you are setting `project` on the bucket resource.  I'd also remove the "and only" language.